eng/illink.targets

```
@@ -105,8 +103,43 @@
     </ItemGroup>
   </Target>
 
+
+  <PropertyGroup>
+    <ILLinkResourcesSubstitutionIntermediatePath>$(IntermediateOutputPath)ILLink.Resources.Substitutions.xml</ILLinkResourcesSubstitutionIntermediatePath>
+    <GenerateResourcesSubstitutions Condition="'$(GenerateResourcesSubstitutions)' == '' and '$(StringResourcesPath)' != ''">true</GenerateResourcesSubstitutions>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(GenerateResourcesSubstitutions)' == 'true'">
+    <ILLinkSubstitutionsXmls Include="$(ILLinkResourcesSubstitutionIntermediatePath)" />
+  </ItemGroup>
+  
+  <!-- If a library uses string resources, the following target generates a substitution xml that will be embedded on the
+  library so that if a consumer wants to run the linker they can specify a feature switch to strip out all resources
+  from the assembly. -->
+  <Target Name="GenerateResourcesSubstitutionFile"  
+          Condition="'$(GenerateResourcesSubstitutions)' == 'true'"
+          Inputs="'$(MSBuildProjectFullPath)'"
+          Outputs="'$(ILLinkResourcesSubstitutionIntermediatePath)'">
```

```suggestion
          Inputs="$(MSBuildProjectFullPath)"
          Outputs="$(ILLinkResourcesSubstitutionIntermediatePath)">
```
I'm surprised incremental build works with those extra apostrophes.

---

Great catch! I'll fix that.